### PR TITLE
Ascii cast

### PR DESCRIPTION
**Pull request by kris-nova**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/1259)

### CODE REVIEW

**Comment by justinsb** [comment]
LGTM... but ... can you squash and not include the submodule updates?

**Comment by chrislovecnm** [comment]
@kris-nova can you remove the submodule changes?

**Comment by kris-nova** [comment]
@chrislovecnm @justinsb - squashed, and removed the submodule changes

**Comment by kris-nova** [comment]
@k8s-bot aws e2e test this

File: README.md [hunk 1]
```diff
@@ -22,9 +22,12 @@ Check out our [tutorial](/docs/aws.md) for launching a Kubernetes cluster hosted
 on AWS.
```
Comment (chrislovecnm) [inline README.md]: LGTM

**Pull request merged**